### DISCUSSION TITLE
Add API for get command output

### DISCUSSION THREAD

**Discussion by Axiumyuyu** [PaperMC/Paper#11508] [Ideas]
### Is your feature request related to a problem?

 when a player(or Console) execute command, Record the output and provide a method to get the message, I think sometimes this will be useful especially when this command is related to DataPacks, ```/execute```, ```/trigger``` or ```/function``` etc.

### Describe the solution you'd like.

In Class ExecuteCommandEvent, add a mathod like ```getResult()```(or one more setting method ```setResult()```)

### Describe alternatives you've considered.

Using NMS, but i think this usually not recommended.

**Comment by JavierFlores09** [comment]
I mean, doesn't the [AsyncChatCommandDecorateEvent](https://jd.papermc.io/paper/1.21.1/io/papermc/paper/event/player/AsyncChatCommandDecorateEvent.html) serve this purpose?

**Reply by Axiumyuyu** [reply]
Well, there is such a way,I never thought about it.Thank you.

**Reply by Axiumyuyu** [reply]
I will try it after that.
